client/app/telecom/telephony/alias/dashboard/telecom-telephony-alias-dashboard.html

```diff
@@ -0,0 +1,113 @@
+<div class="container-fluid"
+     data-ng-if="!$ctrl.loading">
+    <div class="mb-3">
+        <span class="oui-header_6"
+              data-translate="telephony_alias_presentation_title"></span>
+        <a class="oui-link"
+           data-ng-click="$ctrl.hideDetails = !$ctrl.hideDetails"
+           data-translate="{{ ($ctrl.hideDetails ? 'hide' : 'show') | translate }}">
```

Comment:
It's quite confusing to use both `data-translate` directive and `| translate` filter on the same element.

Also you didn't write the test on the proper way.
It should be: `data-ng-bind="$ctrl.hideDetails ? 'show' : 'hide' | translate"`.

Reply:
Why not using [`<oui-collapsible>`](https://master.ui-kit.ovh/#!/oui-angular/collapsible) element instead?

For instance:

```html
<oui-collapsible class="mb-5"
                 data-heading="{{ ::'telephony_alias_presentation_title' | translate }}"
                 data-expanded="true">
    <p class="oui-paragraph">
        <span data-translate="telephony_alias_presentation_part_one"></span>
        <span data-translate="telephony_alias_presentation_part_two"></span>
    </p>
    <p class="oui-paragraph"
       data-translate="telephony_alias_presentation_part_three">
    </p>
</oui-collapsible>
```